app/assets/javascripts/templates/story.ejs

```diff
@@ -6,6 +6,11 @@
         <span class="estimate estimate-<%= value %> input" data-value="<%= value %>" id="estimate-<%= value %>"></span>
       <% } %>
     <% }); %>
+    <% if (view.isSearchResult) { %>
+      <button type="button" class="btn btn-default locate-btn locate">
```

Comment:
Is this `type` attribute really needed?

Reply:
Yes to prevent the default button behavior, thus preventing it from submitting the form.
as per the [W3 specification](https://www.w3.org/TR/2011/WD-html5-20110525/the-button-element.html#the-button-element)